config/v1/types_infrastructure.go

```diff
@@ -144,6 +148,18 @@ type GCPPlatformStatus struct {
 	Region string `json:"region"`
 }
 
+// OpenStackPlatformStatus holds the current status of the OpenStack infrastructure provider.
+type OpenStackPlatformStatus struct {
+	// apiIntIP is an IP address managed by the OpenStack provider
+	// backing the internal `api-int` record.
+	APIIntIP string `json:"apiIntIP,omitempty"`
```

Comment:
we don't use abbreviations for internal. See https://github.com/openshift/api/blob/master/config/v1/types_infrastructure.go#L65 .  Speaking of which, does this field correspond to this URL in some way?

Reply:
Ah yes. This is an IP address that OpenStack generates which that URL will resolve to.

Platforms without a DNSaaS (openstack, baremetal, ovirt) need to do that. I'll update the name.

Reply:
I've updated the field name & comment. Hope this is clearer.